.github/ISSUE_TEMPLATE/report-issue.md

```diff
@@ -1,10 +1,11 @@
-name: Template for 'Report issue'
- about: For issues reported via 'Report issue' banner
- title: ''
- labels: direct user feedback
- assignees: ''
- 
- ---
+---
+name: Template for 'report issue'
+about: For issues reported via 'Report issue' banner
```

Comment:
```suggestion
about: Report issues in upstream Foreman/Katello documentation
```

same as above: these strings are exposed to the user that opens an issue -> make them more user friendly and less meta.